default_metrics.toml

```diff
@@ -0,0 +1,24 @@
+# This defines the default metrics and statistics calculated for each experiment
+
+[metrics]
+daily = ["unenroll"]
+weekly = ["active_hours", "uri_count", "ad_clicks", "search_count"]
+overall = ["active_hours", "uri_count", "ad_clicks", "search_count"]
+
+[metrics.unenroll.statistics.bootstrap_mean]
+num_samples = 1000
+
+[metrics.view_about_logins.statistics.bootstrap_mean]
+num_samples = 1000
+
+[metrics.active_hours.statistics.bootstrap_mean]
+num_samples = 1000
+
+[metrics.uri_count.statistics.bootstrap_mean]
+num_samples = 1000
+
+[metrics.ad_clicks.statistics.bootstrap_mean]
+num_samples = 1000
+
+[metrics.search_count.statistics.bootstrap_mean]
+num_samples = 1000
```

Comment:
Metrics that are referenced in custom config files need to be defined here and are required to have a default statistical treatment.